services/discovery/src/app.py

```diff
@@ -222,21 +231,25 @@ def get_first_valid_report_definition(valid_report_definitions, default=None):
     return valid_report_definitions[0] if any(valid_report_definitions) else default
 
 
-def discover_master_payer_account(world):
-    report_definitions = coeffects_report_definitions(world)
+def get_cur_bucket_if_local(world, report_definitions):
     logger.info(f'Found these ReportDefinitions: {report_definitions}')
     local_buckets = {x['Name'] for x in coeffects_buckets(world)}
     valid_report_definitions = keep_valid(MINIMUM_BILLING_REPORT, report_definitions)
     logger.info(f'Found these _valid_ ReportDefinitions: {valid_report_definitions}')
-    first_valid = get_first_valid_report_definition(valid_report_definitions, default={})
```

Comment:
Forget about valid reports with remote buckets. https://youtu.be/Pw2sex1mJNI?t=9